releasenotes/notes/process-agent-service-tag-extraction-3fb97205155ed1a4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Add service tag extraction based on a process's command line args for service monitoring.
```

Comment:
Need to add configuration details here?

Reply:
Added @maycmlee - mind having another look? 🙇